### PR TITLE
Contract fix

### PR DESCRIPTION
**Pull request by dwalintukan**

### CODE REVIEW

File: src/contracts/oracle.js [hunk 1]
```diff
@@ -20,7 +18,7 @@ const Oracle = {
       return;
     }
 
-    const oracle = getOracleContract(oracleType, contractAddress);
+    const oracle = getContract(oracleType, contractAddress);
     return await oracle.send('invalidateOracle', {
       methodArgs: [],
       gasLimit: 3000000,
```
Comment (frankobe) [inline src/contracts/oracle.js]: @dwalintukan `gasLimit` should be defined as a constant

File: src/contracts/bodhi_token.js [hunk 2]
```diff
@@ -54,12 +52,16 @@ const BodhiToken = {
...
       methodArgs: [owner],
       senderAddress: senderAddress,
     });
   },
 };
 
+function getContract() {
+  return new Contract(Config.QTUM_RPC_ADDRESS, Contracts.BodhiToken.address, Contracts.BodhiToken.abi);
+}
+
```
Comment (chrisli30) [inline src/contracts/bodhi_token.js]: This Contract instance can be a constant on top of this page.

File: src/contracts/centralized_oracle.js [hunk 3]
```diff
@@ -93,12 +91,16 @@ const CentralizedOracle = {
...
       methodArgs: [],
       senderAddress: senderAddress,
     });
   },
 };
 
+function getContract(contractAddress) {
+  return new Contract(Config.QTUM_RPC_ADDRESS, contractAddress, Contracts.CentralizedOracle.abi);
+}
+
```
Comment (chrisli30) [inline src/contracts/centralized_oracle.js]: This Contract instance as well. It's stateless

File: src/contracts/bodhi_token.js [hunk 4]
```diff
@@ -1,8 +1,6 @@
 import Config from '../../config/config';
 import Contracts from '../../config/contracts';
-
-const Qweb3 = require('qweb3');
-const qweb3 = new Qweb3(Config.QTUM_RPC_ADDRESS);
+const Contract = require('qweb3/src/contract');
```
Comment (chrisli30) [inline src/contracts/bodhi_token.js]: Out of curiosity, the reason you stripped Contract out of qweb3 is that we don't need to call new twice?

File: src/contracts/oracle.js (hunk 1 above)
Comment (dwalintukan) [reply src/contracts/oracle.js]: added constants for gas limit for all specified calls.

File: src/contracts/bodhi_token.js (hunk 4 above)
Comment (dwalintukan) [reply src/contracts/bodhi_token.js]: 1. The npm module import couldn't find the Contract class.
2. qweb3 functionality didnt need to be in the Contract instance.
3. extra step for creating qweb3 instance first.

File: src/contracts/bodhi_token.js (hunk 2 above)
Comment (dwalintukan) [reply src/contracts/bodhi_token.js]: good call, fixed

File: src/contracts/centralized_oracle.js (hunk 3 above)
Comment (dwalintukan) [reply src/contracts/centralized_oracle.js]: this one cannot be a constant since I am passing in contractAddress.

Comment (chrisli30) [reply src/contracts/centralized_oracle.js]: right, my mistake

**Pull request merged**